proxy.go

```diff
@@ -64,6 +66,8 @@ func getAllEnvVariables() {
 	// Get the AWS credentials
 	awsRegion = getEnvOrDefault("AWS_REGION", "eu-west-1", false)
 	awsBucket = getEnvOrDefault("AWS_BUCKET", "", true)
+	proxy = getEnvOrDefault("HTTP_PROXY", "", false)
```

Comment:
I think we can do one better—go provides `ProxyFromEnvironment`: https://golang.org/pkg/net/http/#ProxyFromEnvironment

I'll update this shortly.